### ISSUE TITLE
Can not enable German spell checker

### ISSUE THREAD

**Issue by akalt** [Jason3S/vscode-cspell-dict-extensions#2]
When I trie to enable the German spell checker via *"Enable German Spell Checker Dictionary"* I get the error: 

**command 'cSpellExt_german.enableGerman' not found**

I tried several uninstalls/installs of the extension but with no luck. The *"Spell Checker Configuration Info"* shows that the German language is installed and activated globally. Still, the spell checker does not recognize german words.

Even after adding *"cSpell.language": "en,de"* to the VS Code configuration file, the spell checker does not work for German words.

**Comment by Jason3S** [comment]
Thank you for the feedback.

Can you try restarting VSCode?

What do you see when you do the following:
`F1` `Show Spell Checker Configuration Info`
or
Right click in the editor and choose: `Show Spell Checker Configuration Info`

I see something like this:
![image](https://user-images.githubusercontent.com/3740137/30904959-c588c168-a373-11e7-88f7-f9665bf301f2.png)

**Comment by akalt** [comment]
Thanks for the fast response.

I rebooted the Mac installed the "German - Code Spell Checker" which installes the "Spelling Checker for Visual Studio Code" dependency. After doing so I see the following:

![spellchecker_nogerman](https://user-images.githubusercontent.com/1479322/30957501-ba1441c8-a43a-11e7-83ae-7f43af5cc842.png)

At this point I can not enable the German spell checker with F1 *"Enable German Spell Checker Dictionary"* – this gives me the already mentioned error. After adding the *de* to the *user settings* file: "cSpell.language": "en,de" I get this info:

![spellchecker_german](https://user-images.githubusercontent.com/1479322/30957668-343278da-a43b-11e7-9634-b05d29bb070f.png)

**BUT** the spell checker for German does still not work; the Enlish spell checker works just fine.

**Comment by Jason3S** [comment]
It should have shown up on its own without you needing to do anything.  The Language screen is new and I haven't had the chance to update the readme.

Can you un-install the German extension and reinstall it? But do not uninstall the Spell checker, just the German extension.

**Comment by Jason3S** [comment]
I realize you have un-installed it and re-installed it a few times already. I'm just trying to figure out the issue.

**Comment by akalt** [comment]
OK, I tried that, but with no luck. 

<img width="970" alt="vscode_spellchecker_error" src="https://user-images.githubusercontent.com/1479322/31074647-eb5c8e68-a771-11e7-9077-7fe7470b714c.png">

**Comment by Jason3S** [comment]
@akalt did you ever get it to work?

**Comment by akalt** [comment]
No. I really tried everything I can think of. Just tried it again right now - no luck. I guess I should uninstall all the extensions, but, to be honest, I'm to lazy for that :)

**Comment by Jason3S** [comment]
It is acting like the German extension isn't fully installed.

**Comment by akalt** [comment]
I am happy to report that the German extension works now. There is an awesome little button in VS Code that reads "Disable All Installed Extensions" :) ; and that is what I did. After disabling all extensions I installed the spell checker, and know everything works fine - as far as I can tell after a view test.

So, the problem was with one of the other installed extensions.

**Comment by Jason3S** [comment]
I am happy to hear that!
